src/spec.ts

```diff
@@ -207,6 +206,10 @@ export function isHConcatSpec(spec: GenericSpec<GenericUnitSpec<any, any>>): spe
   return spec['hconcat'] !== undefined;
 }
 
+export function autoSizeFitSpec(spec: TopLevelExtendedSpec) {
```

Comment:
This method is weirdly named and it seems unclear to me why we need it. 

We can just call `autoSizeFit(spec.autosize)` directly.

Reply:
Done